### PR TITLE
New package: borgmatic-1.2.0

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/borgmatic/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+version=1.2.0
+revision=1
+build_style=python3-module
+pycompile_module="borgmatic"
+hostmakedepends="python3-setuptools python3-devel"
+makedepends="${hostmakedepends/python3-setuptools/}"
+depends="python3-ruamel.yaml python3-pykwalify"
+short_desc="Simple wrapper script for the Borg backup software"
+maintainer="Renato Aguiar <renato@renag.me>"
+license="GPL-3.0"
```
Comment (abenson) [inline srcpkgs/borgmatic/template]: GPL-3.0-only

File: srcpkgs/python3-pykwalify/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+revision=1
+wrksrc=pykwalify-${version}
+build_style=python3-module
+pycompile_module="pykwalify"
+hostmakedepends="python3-setuptools python3-devel"
+makedepends="${hostmakedepends/python3-setuptools/}"
+depends="python3-yaml python3-docopt python3-dateutil"
+short_desc="Python YAML/JSON schema validation library"
+maintainer="Renato Aguiar <renato@renag.me>"
+license="custom"
```
Comment (abenson) [inline srcpkgs/python3-pykwalify/template]: Looks like MIT

Comment (renatoaguiar) [reply srcpkgs/python3-pykwalify/template]: I also thought that, but it has an extra statement ("The Software shall be used for Good, not Evil.") that seems to make it nonfree: https://www.gnu.org/licenses/license-list.html#JSON

Comment (abenson) [reply srcpkgs/python3-pykwalify/template]: I think you're right -- and that is the JSON license. In that case it should be JSON https://spdx.org/licenses/JSON.html

**Pull request merged**